sharing/README.md

```diff
@@ -0,0 +1,208 @@
+
+### 3. Selecting the Direction of the Rule
+
+The third step involves selecting the direction in which the rule should be applied. If we imagine all the ODM data as a set of spreadhseets, with each sheet containing the data for a table, then there are two ways in which a rule can be applied:
+
+1. It can applied by going through each column in the spreadsheet and filtering out those columns that meets the rule's specification
+2. It can be applied by going through each row in the spreadsheet and filtering out those rows that meets the rule's specification
+
+This step uses the `direction` column which accepts one of two values, `column` to apply the rule column by column or `row` to apply the rule row by row.
+
+### 4. Selecting the Values to Filter Out
+
+The final step involves inputting the values of the selected entities to filter out. This step uses the `ruleValue` column whose values depends on the data type (boolean, numeric etc.) and the statistical type (continuous or categorical) of the variable.
+
+For the statistical type:
+
+- **Continuous Type**: For continuous entities, this column can either specify an interval with a lower and upper limit, a specific value, or a combination of the two. We use the mathematical notation to define an [interval](<https://en.wikipedia.org/wiki/Interval_(mathematics)>). Examples are:
+
+  - [1, 2]: All values between 1 and 2, inclusive of 1 and 2 will be filtered out
+  - (1, 2]: All values between 1 and 2, only inclusive of 2 will be filtered out
+  - [1, 2): All values between 1 and 2, only inclusive of 1 will be filtered out
+  - (1, 2): All values between 1 and 2, excluding 1 and 2 will be filtered out
+  - [Inf, 1]: All values between negative infinity and 1, including 1 will be filtered out
+  - [1, Inf]: All values between 1 and positive infinity, including 1 will be filtered out
+
+  The **Inf** keyword can be used to specify infinity either as the lower or upper bound. Combination of values within this column can be entered in by using the **;** symbol again. For example, [1, 2]; (7, 8); 9
+
+- **Categorical Type**: For categorical entities, this column should specify the category values to filter out seperated one again by the **;** symbol
+
+Users can also use the **all** keyword to filter our all values in an entity.
+
+For the data type, please refer to the [ODM metadata](https://github.com/Big-Life-Lab/ODM/blob/main/metadata_en.md#entity-relationship-diagram) on the allowable value for each one.
+
+Some examples are given below,
+
+1. Exclude values of `rawWW` or `swrSed`
+
+   | ruleValue    |
+   | ------------ |
+   | rawWW;swrSed |
+
+2. Exclude values between March 1st 2021 and December 1st 2021 including the endpoints. All Datetime values inside list must go within double or single quotes.
```

Comment:
The example below doesn't have quotes - is that a typo or does this refer to something else?

Reply:
That's a mistake, I'll update it.